locksmith/src/operations/checkoutConfigOperations.ts

```diff
@@ -55,3 +126,81 @@ export const getCheckoutConfigById = async (id: string) => {
   }
   return null
 }
+
+/**
+ * Checks if a user is authorized to manage a specific checkout config
+ * @param userAddress - The address of the user
+ * @param configId - The ID of the checkout configuration
+ * @returns A boolean indicating whether the user is authorized
+ */
+export const isAuthorizedToManageConfig = async (
+  userAddress: string,
+  configId: string
+): Promise<boolean> => {
+  const checkoutConfig = await CheckoutConfig.findOne({
+    where: {
+      id: configId,
+    },
+  })
+  if (!checkoutConfig) {
+    return false
+  }
+  return isUserAuthorized(userAddress, checkoutConfig)
+}
+
+/**
+ * Retrieves all checkout configurations a user is authorized to manage
+ * @param userAddress - The address of the user
+ * @returns An array of authorized checkout configurations
+ */
+export const getAuthorizedCheckoutConfigs = async (userAddress: string) => {
+  const allConfigs = await CheckoutConfig.findAll({
+    order: [['updatedAt', 'DESC']],
+  })
```

Comment:
Hum. This will not work well, because there are already a few thousand checkout configs... and iterating thru them will take too much time.
I am not sure there is a way to list configs that a user is allowed to manage right now.

I think the only way to handle this would be on the front-end to let a user enter a checkout config id and then verify that they can indeed manage that specific config id.